language/python/develop.md

```diff
@@ -141,7 +141,7 @@ if __name__ == "__main__":
   app.run(host ='0.0.0.0')
 ```
 
-We’ve added the MySQL module and updated the code to connect to the database server, created a database and table. We also created a couple of routes to save widgets and fetch widgets. We now need to rebuild our image so it contains our changes.
+We’ve added the MySQL module and updated the code to connect to the database server, create a database and table. We also created a couple of routes to save widgets and fetch widgets. We now need to rebuild our image so it contains our changes.
```

Comment:
This should be create and not created.

Reply:
What will create the database and table? If it's "we", then sure, "we created", but then I believe the "and" and commas are incorrect.

```diff
- "We've added the MySQL module and updated the code [...], created a database and table."
+ "We've added the MySQL module, updated the code [...], and created a database and table."
```

But if it's the updated code that'll create the database and table, then I think should be "create", because "We've [...] updated the code to connect to the database [...], create a database and table."